lib/tags/index.js

```diff
@@ -18,11 +18,14 @@ export default class TagContainer extends Component {
     const {
       tags,
       removeTag,
-      resetColour
+      resetColour,
+      resetBackgroundColour,
+      resetBorderColour
     } = this.props;
     const resetButtonStyle = {
       color: resetColour,
-      border: '1px solid' + resetColour
+      backgroundColor: resetBackgroundColour,
+      border: '1px solid' + resetBorderColour
     };
```

Comment:
Then I would probably add several CSS clases, and use one or another as props, but we would still have style and logic/structure decoupled.